### ISSUE TITLE
Problem with assets on heroku

### ISSUE THREAD

**Issue by gabrielhilal** [errbit/errbit#806]
I have followed the [docs] (https://github.com/errbit/errbit/blob/master/docs/deployment/heroku.md) to deploy `errbit` on `heroku`, however for some reason it's not loading the assets (404):
The mongo database is working fine, as I can create/remove users. Also, I can see that assets have been successfully precompiled and available in the public folder.
Please note the heroku logs:

>2015-01-14T12:48:45.806414+00:00 heroku[router]: at=info method=GET path="/" host=errbit-real.herokuapp.com request_id=351c28bf-41c1-4308-b148-f1d7f1a21a53 fwd="179.176.185.246" dyno=web.1 connect=1ms service=26ms status=304 bytes=373
2015-01-14T12:48:46.077817+00:00 heroku[router]: at=info method=GET path="/assets/application-537d72cd66bb2da9b88e7f39029a4ef4.css" host=errbit-real.herokuapp.com request_id=3eeaf8f3-cb5f-44f5-938d-f6496670df66 fwd="179.176.185.246" dyno=web.1 connect=2ms service=80ms status=404 bytes=5573
2015-01-14T12:48:46.406997+00:00 heroku[router]: at=info method=GET path="/assets/application-79e3a5026113f4ba59af151c14f3fe95.js" host=errbit-real.herokuapp.com request_id=9b766d9b-2500-41a4-8968-aa889e76d7a3 fwd="179.176.185.246" dyno=web.1 connect=4ms service=250ms status=404 bytes=5572
2015-01-14T12:48:46.779301+00:00 heroku[router]: at=info method=GET path="/" host=errbit-real.herokuapp.com request_id=e12d1e8c-1d32-49c4-84c9-75812c5fda3e fwd="179.176.185.246" dyno=web.1 connect=1ms service=145ms status=304 bytes=373
2015-01-14T12:48:47.043797+00:00 heroku[router]: at=info method=GET path="/assets/application-537d72cd66bb2da9b88e7f39029a4ef4.css" host=errbit-real.herokuapp.com request_id=7289045b-7a59-4658-b22c-fb66ace4a2fb fwd="179.176.185.246" dyno=web.1 connect=1ms service=72ms status=404 bytes=5573
2015-01-14T12:48:47.358885+00:00 heroku[router]: at=info method=GET path="/assets/application-79e3a5026113f4ba59af151c14f3fe95.js" host=errbit-real.herokuapp.com request_id=a122f51b-bb6a-4231-ab61-10d1c0fbff0e fwd="179.176.185.246" dyno=web.1 connect=1ms service=76ms status=404 bytes=5572

I have seen some issues related to a missing `.yml` database file or even setting `user-env-compile` (not available on heroku anymore), but I could not solve the problem.
Any idea?

**Comment by gabrielhilal** [comment]
I have solved the problem adding the `rails_12factor` gem

**Comment by stevecrozz** [comment]
@gabrielhilal We've had some fun with this gem before: https://github.com/errbit/errbit/pull/765

It may be fine to add it now, but I have another idea that I think might be better. This gem accomplishes two things. 1) sets logging to STDOUT and 2) allows the app to serve static assets. My current thinking is these two settings should just be settable using environment variables whether or not you use heroku.

So in other words, I think we should make these settings toggleable with environment variables using something like SERVE_STATIC_ASSETS and LOG_TO and then we just document that these should be set to true and STDOUT for heroku.

**Comment by matiaskorhonen** [comment]
Forgive me if I'm wrong, but would it be possible to add the `rails_12factor ` gem with `require: false` and then require it in an initializer (or somewhere else in the app) if the `HEROKU` environment variable is set?

**Comment by stevecrozz** [comment]
Looks like the rails_12factor gem is back in, so this should be resolved.
